src/dispatchers/simple.coffee

```diff
@@ -3,11 +3,13 @@ Context = require("../context")
 
 module.exports = class SimpleDispatcher
 
-  constructor: (@service, @handlers) ->
-    for resource, actions of @service.default_handlers
-      handlers[resource] ||= {}
+  constructor: (@service, handlers) ->
+    @handlers = @service.default_handlers
+    
+    for resource, actions of handlers
+      @handlers[resource] ||= {}
       for name, handler of actions
-        handlers[resource][name] = handler
```

Comment:
Definitely a bug, but wouldn't it be simpler just to use an `||=` when assigning the default handler to `handlers[resource][name]` ?